src/library/Build.js

```diff
@@ -166,7 +153,6 @@ function buildCustomQuery(events, groupBy, orderBy, limit, offset) {
   // build the event query
   for (let i = 1; i < events.length; i++) {
     const event = events[i];
-
```

Comment:
The section above that validates the number of columns will need to be adjusted. The logic is still setup for when selects and filters were both of size 3.

What we can probably do is just remove this check altogether:
```js
if ((header.length - 1) % 3 !== 0) {
  throw new Error(`Expecting number of columns to be divisible by 3 plus 1, found ${header.length} columns total`);
}
```

Since a more comprehensive check is already being done inside `extractSelectFilterCounts()`.

![image](https://user-images.githubusercontent.com/5861379/102017886-f17b7000-3dac-11eb-904b-19f97278dcfd.png)

Reply:
Sorry. I found the same issue yesterday. I tested complicated filters and worked but simple one didn't work 😓 I will let you know once I fixed this part.

Reply:
@shoenseiwaso I have fixed you mentioned and other expected issues

> ![image](https://user-images.githubusercontent.com/5861379/102017886-f17b7000-3dac-11eb-904b-19f97278dcfd.png)

FYI, data should be as follows
![image](https://user-images.githubusercontent.com/16159767/102187908-f436ac00-3ef7-11eb-9b44-5e39ff04fa99.png)